src/20230925_AaveV2_Eth_AaveTreasuryRWAAllocationPartI/AaveV2_Ethereum_AaveTreasuryRWAAllocationPartI_20230925.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveMisc} from 'aave-address-book/AaveMisc.sol';
+import {AaveV2Ethereum, AaveV2EthereumAssets} from 'aave-address-book/AaveV2Ethereum.sol';
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {SafeERC20} from 'solidity-utils/contracts/oz-common/SafeERC20.sol';
+
+/**
+ * @title Aave treasury RWA Allocation Part I
+ * @author Marc Zeller - Aave-Chan Initiative
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x71db494e4b49e7533c5ccaa566686b2d045b0761cb3296a2d77af4b500566eb0
+ * - Discussion: https://governance.aave.com/t/arfc-aave-treasury-rwa-allocation/14790
+ */
+contract AaveV2_Ethereum_AaveTreasuryRWAAllocationPartI_20230925 {
+  using SafeERC20 for IERC20;
+
+  address public constant CENTRIFUGE = 0x9c489E4efba90A67299C1097a8628e233C33BB7B; // Placeholder until address is confirmed, DO NOT MERGE
+  uint256 public constant STREAM_AMOUNT = 500e18;
+  uint256 public constant STREAM_DURATION = 720 days;
+  uint256 public constant ACTUAL_STREAM_AMOUNT_AAVE =
```

Comment:
to have consistent naming I would remove _AAVE

Reply:
Updated & fixed